vm/src/frame.rs

```diff
@@ -562,7 +564,7 @@ impl Frame {
             }
             bytecode::Instruction::MakeFunction { flags } => {
                 let _qualified_name = self.pop_value();
-                let code_obj = self.pop_value();
```

Comment:
I think this should panic (i.e. `self.pop_value().downcast().expect(...) `) rather than raise a `TypeError`, since a non-code object on the stack here would indicate a compiler bug.